public/styles.css

```diff
@@ -1,3 +1,61 @@
-body {
-  margin: 0;
+#main {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+.mainBox {
+  flex-direction: column;
+  /*align-items: center;*/
+  justify-content: space-around;
+  display: flex;
+  flex-wrap: wrap;
+  align-self: flex-start;
+}
+.dtborder {
+  position: relative;
+  border: 20px solid #ffecb7;
+  background: #ffffff;
+  padding: 60px;
+  text-align: center;
+  height: 100px;
+  line-height: 100px;
+  width: 100px;
+}
+.dtborder:before {
+  content: "";
+  position: absolute;
+  top: 0px;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  border: 20px solid sandybrown;
+}
+
+.dtborder:after {
+  content: "";
+  position: absolute;
+  top: 20px;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+  border: 20px solid #b7ecb7;
+}
+.data {
```

Comment:
staraj się nazywać trochę bardziej rzeczowo elementy html. Najlepiej nazywać je zgodnie z domeną której dotyczą oraz funkcję której pełnią. W CSS panuje konwencja kebab case w większości, ale np w reatcie często się używa CamelCase. Także wystarczy żebyś przestrzegał jednej dowolnej, wygodnej dla Ciebie. Np. tutaj jest to funkcjonalnie input width i height, ale nie mamy domeny, bo ta aplikacja nie posiada logiki biznesowej. Dlatego przyjmyjmy za domenę holder tego inputa czyli box. Zatem box-input-width i box-input-height, albo jak nie potrzebuejsz ich rozróżniac to po prostu box-input ;)